src/libcore/char.rs

```diff
@@ -263,6 +264,8 @@ pub trait CharExt {
     fn escape_unicode(self) -> EscapeUnicode;
     #[stable(feature = "core", since = "1.6.0")]
     fn escape_default(self) -> EscapeDefault;
+    #[unstable(feature = "char_escape", issue = "0")]
```

Comment:
This should get filled in with a tracking issue, and could you open up a new issue in lieu of #27791 just to refresh the discussion a bit?